### PR TITLE
Release 5.2.0

### PR DESCRIPTION
**Pull request by fredericDelaporte**
Preparing the release.

I have already moved issues in the 5.2 milestone to a [5.3 milestone](https://github.com/nhibernate/nhibernate-core/milestone/21) when they were non closed and without any pending PR.

16 PR are awaiting in the [5.2 milestone](https://github.com/nhibernate/nhibernate-core/milestone/14). Some are WIP or looks like, others are pending approval.

### CODE REVIEW

**Comment by hazzik** [comment]
@fredericDelaporte can we hold-on the release till end of this week? I want try to finish some WIP tasks.

**Comment by fredericDelaporte** [comment]
No troubles, I was not intending to rush it anyway.

File: releasenotes.txt [hunk 1]
```diff
@@ -61,11 +67,10 @@ Release notes - NHibernate - Version 5.2.0
           operations to be done by their base AfterInitialize will need to be changed
           accordingly.
 
-Not yet merged and not counted: #1836, #1793
```
Comment (fredericDelaporte) [inline releasenotes.txt]: #1793 is not merged, but moved out, although I would rather have it in 5.2.

**Comment by fredericDelaporte** [comment]
Rebased, and release notes are up-to-date.

**Pull request merged**